converter/src/main/java/gov/cms/qpp/conversion/decode/SkeletalSectionDecoder.java

```diff
@@ -0,0 +1,32 @@
+package gov.cms.qpp.conversion.decode;
+
+import gov.cms.qpp.conversion.Context;
+import gov.cms.qpp.conversion.model.Node;
+
+import java.util.Objects;
+
+import org.jdom2.Element;
+
+public abstract class SkeletalSectionDecoder extends QrdaDecoder {
+
+	public static final String CATEGORY = "category";
+
+	private final String category;
+
+	public SkeletalSectionDecoder(Context context, String category) {
+		super(context);
+
+		Objects.requireNonNull(category, "category");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'CATEGORY' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)